workspaces/src/types/block.rs

```diff
@@ -1,11 +1,16 @@
+use near_account_id::AccountId;
 use near_primitives::views::{BlockHeaderView, BlockView};
 
+use crate::types::{Balance, ChunkHeader};
 use crate::{BlockHeight, CryptoHash};
 
 /// Struct containing information on block coming from the network
-#[derive(Clone, Debug, PartialEq)]
+#[derive(Clone, Debug, Eq, PartialEq)]
+#[non_exhaustive]
 pub struct Block {
-    header: BlockHeader,
+    pub author: AccountId,
+    pub header: BlockHeader,
+    pub chunks: Vec<ChunkHeader>,
```

Comment:
@shariffdev Let's also update Block and ChunkHeader types to follow the approach of using getters instead of exposing internal fields (`#[non_exhaustive]` should be removed then)